### DISCUSSION TITLE
Education, critique, and action

### DISCUSSION THREAD

**Discussion by skarakas1** [albertkun/21S-ASIAAM-191A#70] [Past Thinking Caps]
Participatory mapping and critical cartography can be improved by continued promotion of open-source technology and by improving education. Open-source allows anyone to access source code to use or improve it; better education is necessary for more inclusion, rejection of outdated notions, and broader acceptance of diverse perspectives.
In school, I took geography and world history classes, but because they were so broad, and so concentrated on “fact”-based learning and memorization, they were bound to fail by any standard other than a Euro-centric/settler colonist worldview.
I think that in geography, and in K-12 education in general, there is a lack of emphasis on critique and action. Memorization and quizzing is a pretty poor model for education because it favors the reproduction of power-knowledge, rather than creativity, critical thinking, and praxis.
The ideas most in need of critique are often those that are the most generally accepted, or conventional wisdom, and educators should have nothing to fear if the ideas can hold up. One of my favorite maps that Mariah Tso showed was the world map that shows the connection of the world’s ocean. This is a great example of how conventional wisdom can be limiting—we all know the mercator projection but it is just one way of looking at the world, and even with a multi-scalar map like Google Maps, it can not show every possible representation. It de-centers Europe, North, and land masses.
Action is as important as critique—the social sciences will never have experimental controls, and the scientific perspective itself reproduces dichotomies. I have been reading John Friedmann’s essays on urban planning which focus on mutual learning and what he calls transactive planning. There are limits to formal academic practice that cannot substitute for experiential knowledge, and there will never be a time when the perfect theory has been developed, so we must continuously learn, act, and critique to participate in real change.

**Comment by annyrviloria** [comment]
So many important points here! I am in total agreement with the point that the learning models we have in place are not conducive to critical thinking or praxis. It's the same idea of teaching for the test - it helps schools meet certain funding and accreditation standards when their students are good test-takers, yet it disincentivizes students' willingness to challenge the status-quo and to think outside of the box. The interesting part is that we continue to justify standardized testing at schools as an unbiased marker of student success when there's an inherent bias for certain types of learning in these tests...

I also really like the point that "the ideas most in need of critique are often those the are most generally accepted". I want to keep this idea in the back of my mind as we continue going through this course and through my graduate program because it is a good reminder that there's always room to push the boundaries of knowledge, even for the ideas we seem to be most sure about.

**Comment by jsxu315** [comment]
I agree that there needs to be more focus on critique in K-12 education! In high school, I had a history teacher that introduced us to a little bit of critique (I don’t think it was required by the curriculum), and it was really helpful in letting me see the limits of Eurocentric/memorization-based history classes. I think that the media also ties into this, because it reaches parents that influence how school boards vote; right now, learning about these kinds of ideas requires you to be more proactive (which takes up time/energy), and in an ideal world it would be more accessible.
